src/build_info.rs

```diff
@@ -17,8 +17,11 @@ macro_rules! generate_build_info {
             },
             git: balthazar::build_info::GitInfo {
                 branch: env!("VERGEN_GIT_BRANCH"),
+                commit_author_email: env!("VERGEN_GIT_COMMIT_AUTHOR_EMAIL")
```

Comment:
Since this is new. shouldn't the vergen version be bumped on the Cargo.toml?

Reply:
AFAIK the vergen version is specified in the application that uses Balthazar, no?

Reply:
but you are right, we can enforce it here as well :-D, done